package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "react-router-docs",
+  "version": "0.0.1",
+  "description": "Rewrite of the React Router docs - work in progress",
+  "author": "Mateusz Zatorski <mateuszzatorski@gmail.com> (https://github.com/knowbody)",
+  "private": true,
+  "main": "README.md",
+  "scripts": {
+    "add-contributor": "all-contributors add",
```

Comment:
This will give you a prompt to just provide the user's github username and the types of contributions it'll add everything to the rc file for you.